packages/xo-web/src/common/xo/index.js

```diff
@@ -1287,41 +1294,42 @@ export const pauseVms = vms =>
 
 export const recoveryStartVm = vm => _call('vm.recoveryStart', { id: resolveId(vm) })
 
-export const restartVm = async (vm, force = false) => {
-  try {
-    await confirm({
-      title: _('restartVmModalTitle'),
-      body: _('restartVmModalMessage', { name: vm.name_label }),
-    })
-
-    return await _call('vm.restart', { id: resolveId(vm), force })
-  } catch (error) {
-    if (error === undefined) {
-      return
-    }
+export const restartVm = async (vm, hardRestart = false) => {
```

Comment:
Can you mutualize stop/restart method? There are several repetitive lines of code.